.travis.yml

```diff
@@ -4,6 +4,13 @@ jdk: oraclejdk8
 
 env:
   global:
+    - SONAR_HOST_URL="https://sonarcloud.io"
+
+    # SONAR_LOGIN value
+    # please note this value is available for pull requests from the same repository only
+    # see https://docs.travis-ci.com/user/pull-requests/#Pull-Requests-and-Security-Restrictions
```

Comment:
@KengoTODA you can remove SONAR_LOGIN variable from travis-ce project settings. This secret should be enough.
WARNING we have to merge this PR to see how full build with sonar analysis is performed.
I will be online throughout the day. You can find me in telegram as kzaikin for quick fixes should it be needed